sql/custom/world/AntiCheat/2019_06_24_00_ANTICHEAT.sql

```diff
@@ -0,0 +1,15 @@
+-- Strings for ASH + AFH allerts
```

Comment:
well, the name of the sql will be renamed, so it does not matter much, but, better not use capital letters in the file name